lib/ship_fosdick/downloader.rb

```diff
@@ -0,0 +1,21 @@
+module ShipFosdick
+  class Downloader
+    def initialize(bucket)
+      @bucket = bucket
+    end
+
+    def download
+      [].tap do |returned_files|
+        bucket.objects.each do |object|
+          next unless object.key.downcase.include?('shp')
```

Comment:
Further thought: maybe this might even be worth defining a policy-object for (e.g. `IsShippingManifest?`). Then we can totally isolate the knowledge of what is a shipping manifest, thus making it _much_ easier to generalize for other fulfilment services.